README.md

```diff
@@ -26,9 +26,31 @@ The commands below assume you have a shell alias setup in your .bashrc or .zshrc
 -   PHP: Run `./vendor/bin/pint` or adjust your editor to run it on save.
 -   TS: Run `npx prettier -w .` before committing, or adjust your editor to run it on save.
 
+#### These commands _will_ run automatically before each commit, so you technically don't have to worry about it
+
+## Database + Common Problems, Issues and Errors with initial build or setup:
```

Comment:
Should we just call this section FAQ, and then phrase each subsection as a question?